tests/conftest.py

```
@@ -275,6 +310,257 @@ def private_project_committed(db_session: Session, setup_test_user_committed: Us
+        '''"""Helper functions."""
+
+def clamp(value: int, min_val: int, max_val: int) -> int:
+    """Clamp a value to a range."""
+    return max(min_val, min(value, max_val))
+'''
+    )
+
+    # Initialize as a Git repo so clone_or_copy_repository can open it with GitPython
+    import subprocess
+    subprocess.run(
+        ["git", "init"],
```

Ruff (0.15.4)

[error] Starting a process with a partial executable path

(S607)